optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/pages/spotroster/ShiftBlobView.java

```diff
@@ -153,4 +204,32 @@ public void remove() {
     public Blob<OffsetDateTime> getBlob() {
         return blob;
     }
+
+    private void setBlobColorAccordingToScore(HardSoftScore score) {
+        if (score.getHardScore() != 0) {
```

Comment:
hard score can be positive in the future if we ever add hard rewards, so != 0 is a bit risky.
It's safer to use !score.isFeasible() (which checks if hardsScore >= 0)